README.md

```diff
@@ -1,93 +1,104 @@
-    --freeze_base_model \
-    --push_to_hub \
-    --trust_remote_code
-```
+# Parler-TTS
+
+[[Paper we reproduce]](https://arxiv.org/abs/2402.01912)
+[[Models]](https://huggingface.co/parler-tts)
+[[Training Code]](training)
+[[Interactive Demo]](https://huggingface.co/spaces/parler-tts/parler_tts_mini)
+
+> [!IMPORTANT]
+> We're proud to release Parler-TTS v0.1, our first 300M-parameters Parler-TTS model, trained on 10.5K hours of audio data.
+
+Parler-TTS is a reproduction of the text-to-speech (TTS) model from the paper [Natural language guidance of high-fidelity text-to-speech with synthetic annotations](https://www.text-description-to-speech.com)
+by Dan Lyth and Simon King, from Stability AI and Edinburgh University respectively. 
+
+Contrarily to standard TTS models, Parler-TTS allows you to directly describe the speaker characteristics with a simple text description where you can modulate gender, pitch, speaking style, accent, etc.
+
+## Usage
+
+> [!TIP]
+> You can directly try it out in an interactive demo [here](https://huggingface.co/spaces/parler-tts/parler_tts_mini)!
+
+Using Parler-TTS is as simple as "bonjour". Simply use the following inference snippet.
+
+```py
+from parler_tts import ParlerTTSForConditionalGeneration
+from transformers import AutoTokenizer, AutoFeatureExtractor
+import soundfile as sf
 
-Tips:
-1. **Number of labels:** normalisation should be applied to the target class labels to group linguistically similar accents together (e.g. "Southern Irish" and "Irish" should both be "Irish"). This helps _balance_ the dataset by removing labels with very few examples. You can modify the function `preprocess_labels` to implement any custom normalisation strategy.
+model = ParlerTTSForConditionalGeneration.from_pretrained("parler-tts/parler_tts_300M_v0.1")
+tokenizer = AutoTokenizer.from_pretrained("parler-tts/parler_tts_300M_v0.1")
 
-## Step 2: Annotate the Training Set
+prompt = "Hey, how are you doing today?"
+description = "A female speaker with a slightly low-pitched voice delivers her words quite expressively, in a very confined sounding environment with clear audio quality. She speaks very fast."
 
-Annotate the training dataset with information on: SNR, C50, pitch and speaking rate. 
+input_ids = tokenizer(description, return_tensors="pt").input_ids
+prompt_input_ids = tokenizer(prompt, return_tensors="pt").input_ids
+
+generation = model.generate(input_ids=input_ids, prompt_input_ids=prompt_input_ids)
+audio_arr = generation.cpu().numpy().squeeze()
+sf.write("parler_tts_out.wav", audio_arr, model.config.sampling_rate)
+```
 
-## Step 3: Aggregate Statistics
 
-Aggregate statistics from Step 2. Convert continuous values to discrete labels.
+## Installation steps
 
-## Step 4: Create Descriptions
+Parler-TTS has light-weight dependencies and can be installed in one line:
+```sh
+pip install parler-tts
+```
 
-Convert sequence of discrete labels to text description (using an LLM). 
+## Gradio demo
```

Comment:
IMO this is less interesting than the training steps -> the Gradio demo can be done with just the tab at the top of the README, since it's not really the focus of the release.

Instead, I would replace this with a section on `Training`. You can then either put the training steps in the main README here, or add a link to the subfolder on training if you think it will get too long. I would have a preference for the former, since we're actively trying to encourage people to use our training code.